server/game/GameActions/PlayCardAction.js

```diff
@@ -58,6 +64,19 @@ class PlayCardAction extends CardGameAction {
                 context.game.resolveAbility(actionContext);
             });
         });
+
+        playEvent.addChildEvent(
+            context.game.getEvent('unamedEvent', { card, context }, () => {
+                if (playActions.length === 0 && this.revealOnIllegalTarget) {
+                    const message =
+                        this.revealOnIllegalTargetMessage ||
+                        '{0} uses {1} to only reveal and not play {2}';
```

Comment:
What scenario does this cover? the wording isn't right and I need to understand what it's doing to suggest different wording

Reply:
Cards like Chaos Portal and Wormhole Technician I think.

Reply:
Yeah I think I'll defer to what I said on the other PR that introduces something similar. A custom message is overkill. It being generic would be fine

Reply:
Fixed, thanks!